src/translations/en.json

```diff
@@ -30,6 +30,7 @@
     "caseName": "case name",
     "uploadCase": "Upload case",
     "uploadErrorMsg": "Please upload the case file",
+    "uploadFileExceedingLimitSizeErrorMsg": "Select case exceeds size limit ({maxSize} MB).{br}Please compress the file in zip, gzip or bzip2 format then try to upload it again",
```

Comment:
Maybe use 'file' instead of 'case' here for the generic message

Reply:
Fixed